app/services/fact_importer.rb

```diff
@@ -20,7 +20,8 @@ def self.fact_features
 
   def self.authorized_smart_proxy_features
     # When writing your own Fact importer, provide feature(s) of authorized Smart Proxies
-    raise NotImplementedError
+    Rails.logger.debug("Importer #{self} does not implement authorized_smart_proxy_features.")
```

Comment:
I guess delegated methods doesn't get inherited? But anyway, this seems more polite :+1: